packages/components/src/tabs/tabs.tsx

```diff
@@ -163,15 +195,35 @@ const TriggerBase = styled(View, {
       },
     },
     active: {
-      true: {
-        cursor: 'default',
```

Comment:
not having a default cursor in this state would be a regression

Reply:
definitely, thx